src/lib/db/strategy-store.ts

```diff
@@ -157,8 +158,16 @@ export default class StrategyStore implements IStrategyStore {
     }
 
     // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-    async importStrategy(data): Promise<void> {
-        const rowData = this.eventDataToRow(data);
+    async importStrategy(data: IStrategyImport): Promise<void> {
```

Comment:
You can now remove the line above:
`// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types`